.github/workflows/license_check.yml

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: "Run License Check"
```

Comment:
Should we consider the `apache/skywalking-eyes@main` Github action which Apache ORC currently uses: https://github.com/apache/orc/blob/main/.github/workflows/build_and_test.yml#L178-L189

It looks simpler to use IMHO.

Reply:
Yes, that looks much simpler indeed. Question, do you know why it needs the Github token?

Reply:
It seems required to comment on the PR:
https://github.com/apache/skywalking-eyes/blob/3ea9df11bb3a5a85665377d1fd10c02edecf2c40/pkg/review/header.go#L62-L64

Reply:
That's fancy, let me change to Skywalking!

Reply:
Done 👍